docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,396 @@
+- `device name`: The name of the camera
+
+For example:
+```shell
+curl -X PUT -d '{
+    "StartStreaming": {
+      "InputImageSize": "640x480",
+      "OutputVideoQuality": "5"
+    }
+}' http://localhost:59882/api/v2/device/name/<device name>/StartStreaming
+```
+
+Supported Input options:
+- `InputFps`: Ignore original timestamps and instead generate timestamps assuming constant frame rate fps. (default - same as source)
+- `InputImageSize`: Specifies the image size of the camera. The format is `wxh`, for example "640x480". (default - automatically selected by FFmpeg)
+- `InputPixelFormat`: Set the preferred pixel format (for raw video). (default - automatically selected by FFmpeg)
+
+Supported Output options:
+- `OutputFrames`: Set the number of video frames to output. (default - no limitation on frames)
+- `OutputFps`: Duplicate or drop input frames to achieve constant output frame rate fps. (default - same as InputFps)
+- `OutputImageSize`: Performs image rescaling. The format is `wxh`, for example "640x480". (default - same as InputImageSize)
+- `OutputAspect`: Set the video display aspect ratio specified by aspect. For example "4:3", "16:9". (default - same as source)
+- `OutputVideoCodec`: Set the video codec. For example "mpeg4", "h264". (default - mpeg4)
+- `OutputVideoQuality`: Use fixed video quality level. Range is a integer number between 1 to 31, with 31 being the worst quality. (default - dynamically set by FFmpeg)
+
+
+### Determine Stream Uri of Camera
+The device service provides a way to determine the stream URI of a camera.
+
+Query parameter:
+- `device name`: The name of the camera
+
+```bash
+curl -s http://localhost:59882/api/v2/device/name/<device name>/StreamURI | jq -r '"StreamURI: " + '.event.readings[].value''
+```
+
+The response to the above call should look similar to the following:
+
+```
+StreamURI: rtsp://localhost:8554/stream/NexiGo_N930AF_FHD_Webcam__NexiG-20201217010
+```
+
+### Play the RTSP stream. 
+
+   mplayer can be used to stream. The command follows this format: 
+   
+   `mplayer rtsp://<IP address>:<port>/<streamname>`.
+
+   Using the `streamURI` returned from the previous step, run mplayer:
+   
+   ```bash
+   mplayer rtsp://localhost:8554/stream/NexiGo_N930AF_FHD_Webcam__NexiG-20201217010
+   ```
+
+  - To shut down mplayer, use the ctrl-c command.
+### Stop Video Streaming
+To stop the usb camera from live streaming, use the following command:
+
+Query parameter:
+- `device name`: The name of the camera
+
+For example:
+```shell
+curl -X PUT -d '{
+    "StopStreaming": "true"
+}' http://localhost:59882/api/v2/device/name/<device name>/StopStreaming
+```
+## Shutting Down
+To stop all EdgeX services (containers), execute the `make down` command:
+
+1. Navigate to the `edgex-compose/compose-builder` directory.
+
+   ```shell
+   cd ~/edgex/edgex-compose/compose-builder
+   ```
+1. Run this command
+   ```bash
+   make down
+   ```
+1. To shut down and delete all volumes, run this command
```

Comment:
Make a note this is optional and a warning that this will delete all edgex-related data.